src/index.js

```diff
@@ -39,12 +40,13 @@ function renderTasks() {
     checkbox.classList.add('check-box');
     checkbox.type = 'checkbox';
     checkbox.index = task.index;
-    // checkbox.checked = task.completed;
+    checkbox.checked = task.completed;
 
     // checkbox.addEventListener('change', () => {
     //   task.completed = checkbox.checked;
     //   save();
```

Comment:
- [ ] For best practices and clean code reasons, kindly ensure that you remove all the commented and unused code lines from your code base